docs/api-reference/geo-layers/mvt-layer.md

```diff
@@ -119,14 +119,13 @@ On top of the [default options](/docs/api-reference/core/layer.md#loadoptions),
 
 ##### `binary` (Boolean, optional)
 
-* Default: false
+* Default: true
 
-Use tile data in [binary format](https://github.com/visgl/loaders.gl/blob/master/modules/gis/docs/api-reference/geojson-to-binary.md) to improve performance. It removes the need for serialization and deserialization of data transferred by the worker back to the main process.
+Use tile data in [binary format](https://github.com/visgl/loaders.gl/blob/master/modules/gis/docs/api-reference/geojson-to-binary.md) to improve performance (2-3x faster on large datasets). It removes the need for serialization and deserialization of data transferred by the worker back to the main process. 
```

Comment:
About the link 
- For "binary data" I think we should link to a deck.gl page on the topic.
- For the details on "performance", I'd suggest referencing the loaders.gl MVTLoader page and explain the advantages there.
- Question for the current link: are we actually calling ` GeoJSONToBinary` in the MVTLoader? Or are we loading directly into binary? The link is also into the git tree rather than into the loaders.gl docs. 

```suggestion
Load tile data in [binary format](https://github.com/visgl/loaders.gl/blob/master/modules/gis/docs/api-reference/geojson-to-binary.md) to improve performance. 
```

@felixpalmer I realize improving the docs across two frameworks represents a lot of work, but ultimately good documentation is a big part of the user value of a complex feature like this. I am not intending for this to be a never ending task, so make your own calls on what makes sense and how far you want to take it.